src/src/DataStructs/NodeStruct.cpp

```diff
@@ -1,5 +1,6 @@
 #include "../DataStructs/NodeStruct.h"
 
+#ifdef FEATURE_ESP_P2P
 const __FlashStringHelper * getNodeTypeDisplayString(uint8_t nodeType) {
```

Comment:
These things should not be removed as they are used in other places too.
For example C009 is now failing to build.

Reply:
I will look into it further. any help would be appreciated as the code is not documented around these pieces.
I know is old code but would be nice to FEATURE flag as much as possible for the flexibility.

I was thinking that in the end would be 3 flags: FEATURE_UDP ; FEATURE_SYSLOG; FEATURE_ESP_P2P. I started with this one as my expectation was that this is the most flash usage hungry one.

If you agree then I could start looking into these 3 flags where they will be used in layers: FEATURE_UDP will be required by either one and all the specific plugins including C009